src/components/distributions/DistributionEditForm.jsx

```diff
@@ -15,15 +15,23 @@ function DistributionEditForm({ path, type }) {
   return (
     <Box sx={{ mt: 12, mx: 'auto', width: '40rem' }}>
       <Typography variant="h2" sx={{ textAlign: 'center', }}>{`${gym?.name}'s ${type}`}</Typography>
-      <form id="distribution-form" style={{ display: 'flex', flexDirection: 'row', flexWrap: 'wrap', justifyContent: 'center', padding: '1rem', width: '30rem' }}>
+      <form
+        id="distribution-form"
```

Comment:
Is the id of distribution-form needed still?

Reply:
There is a css file still in use, another thing planned is either removing the css file or move all the `sx` styling into the css file.